logging/README.md

```diff
@@ -105,7 +115,7 @@ After you create the location for your customization files, you can customize th
 my-viya4mon-user-dir/user.env
 
 my-viya4mon-user-dir/logging/user.env
-my-viya4mon-user-dir/logging/user-values-elasticsearch-open.yaml
+my-viya4mon-user-dir/logging/user-values-opensearch-open.yaml
```

Comment:
File name should be: my-viya4mon-user-dir/logging/user-values-opensearch.yaml

Reply:
Correction made.